docs/GettingStarted.asciidoc

```diff
@@ -201,6 +201,42 @@ of the openQA security model, refer to the
 http://lizards.opensuse.org/2014/02/28/about-openqa-and-authentication/[detailed
 blog post] about the subject by the openQA development team.
 
+
+=== Job groups
+A job can belong to a job group. Those job groups are displayed on the index page
+and in the +Job Groups+ menu on the navigation bar. From there the job group overview
+pages can be accessed. Besides the test results the job group overview pages provide
+a description about the job group and allow commenting.
+
+Job groups have (mostly cleanup related) properties. The configuration can be done
+in the operators menu for job groups.
+
+It is also possible to put job groups into categories. The nested groups will then
+inherit properties from the category. The categories are meant to combine job groups
+with common builds so test results for the same build can be shown together on
+the index page.
+
+
+=== Cleanup
+The following cleanup settings can be done on job-group-level:
+
+* size limit: limits size of assets
+* keep logs for: specifies how long logs of a non-important job are retained after
+  it finished
+* keep important logs for: how long logs of an important job are retained after it
+  finished
+* keep results for: specifies how long results of a non-important job are retained
+  after it finished
+* keep important results for: how long results of an important job are retained after
+  it finished
+
+Note that deletion of job results includes deletion of logs and will cause the job to
```

Comment:
you could use the *WARNING:* macro, and change it to "The deletion..." or even "The cleanup of...".